firebase.json

```diff
@@ -0,0 +1,45 @@
+{
+  "hosting": {
+    "target": "angeleno-my-account-sandbox-422814",
+    "public": "build/web",
+    "ignore": [
+      "firebase.json",
+      "**/.*",
+      "**/node_modules/**"
+    ],
+    "rewrites": [
+      {
+        "source": "/maps/**",
+        "function": "maps",
+        "region": "us-central1"
+      },
+      {
+        "source": "/auth0/**",
+        "function": "auth0",
+        "region": "us-central1"
```

Comment:
Probably question on the infra repo, but is there a reason why we're using us-central1. Our preference is us-west1 (Oregon, etc.)

Reply:
I did not specify a region, so it defaulted to `us-central1`

Reply:
To be changed to us-west1